api/envoy/service/ext_proc/v3/external_processor.proto

```diff
@@ -332,3 +344,16 @@ message BodyMutation {
     bool clear_body = 2;
   }
 }
+
+// This message specifies a new timer duration and the original
+// request direction. If these fields are not configured correctly,
+// the response message will be ignored by the Envoy ext_proc filter.
+message NewProcessingTimeout {
+  // Specifies the new timer duration.
+  // Has to be >= 1ms and <= 10000ms.
+  google.protobuf.Duration message_timeout = 1;
```

Comment:
nit: Please add a PGV rule to enforce this: https://github.com/bufbuild/protoc-gen-validate#:~:text=google.protobuf.Duration%20x%20%3D%201%20%5B(validate.rules).duration%20%3D%20%7B%0A%20%20%20%20gte%3A%20%7B%7D%2C%0A%20%20%20%20lt%3A%20%20%7Bseconds%3A%201%7D%0A%20%20%7D%5D%3B

Reply:
Tried added PGV rule, something like: 
 (validate.rules).duration = {
      lt {seconds: 10}
      gte {nanos: 1000000}
    },
 however it does not work. i.e, even config duration value out-of-bounds(, like 20s), it doesn't trigger configuration failure, no exception thrown.  I guess it is because this is on working thread. There should be no exception thrown.